content/doc/administration/requirements/jenkins-on-java-17.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: documentation
+title: Running Jenkins on Java 17
+---
+
+For simplicity, this document describes how to run the most recent version of Jenkins on Java 17.
+
+NOTE: Production deployments should use only LTS versions of Jenkins.
```

Comment:
Is this documented elsewhere? It’s not the official recommendation as far as I know.

Reply:
@timja good question. I've always thought (and recommended) that LTS would be the recommended way. What would be the reasons to NOT recommend it?

Reply:
Drives people aways from the weekly line.

This is the official wording: https://www.jenkins.io/download/weekly/

A Java 17 PR should not introduce this change

Reply:
👍  thanks for the explanantion